### ISSUE TITLE
What is your Alfred hotkey

### ISSUE THREAD

**Issue by erusev** [sindresorhus/ama#567]
For launcher:
![image](https://user-images.githubusercontent.com/184170/50041224-35bac500-005a-11e9-9dfa-88646f56e49b.png)

and for clipboard history:
![image](https://user-images.githubusercontent.com/184170/50041231-48cd9500-005a-11e9-9512-5ba43dbfbaf9.png)

**Comment by erusev** [comment]
Do you use another app for clipboard history?

**Comment by sindresorhus** [comment]
No, I never saw the value of a clipboard history. Things I'll need again, I store properly.
